include/sdf/Root.hh

```diff
@@ -152,7 +152,7 @@ namespace sdf
     public: sdf::ElementPtr Element() const;
 
     /// \brief Private data pointer
-    IGN_UTILS_UNIQUE_IMPL_PTR(dataPtr)
+    IGN_UTILS_IMPL_PTR(dataPtr)
```

Comment:
We'll need to define the copy ctor and copy assignment operator to properly handle frame graphs.

Reply:
to verify that frame graphs work, we would need a test that loads an `sdf::Root` from an SDFormat string or file and then copies it and tries to resolve a pose or attached-to frame to confirm that the graphs are properly copied

Reply:
I think I'm missing something here. I added tests to capture frame copying, but the tests just passed without requiring the definition of a copy constructor and copy assignment operator. See [6b64370](https://github.com/gazebosim/sdformat/pull/844/commits/6b6437074cd8522543cc20152dc914e0743d1ce9).

Reply:
you're right; I commented this before I saw your latest commit; this is fine